Dockerfile

```diff
@@ -119,11 +120,14 @@ ENV DEV_APT_DEPS=${DEV_APT_DEPS} \
 # Note missing man directories on debian-buster
 # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
 # Install basic and additional apt dependencies
-RUN mkdir -pv /usr/share/man/man1 \
+RUN apt-get update \
+    && apt-get install --no-install-recommends -yqq apt-utils >/dev/null 2>&1 \
+    && apt-get install -y --no-install-recommends curl gnupg2 \
```

Comment:
We already have installed `gnupg`. Is it not enough?
https://github.com/apache/airflow/blob/7a260e53ee5741dd00b5da7972200204ccec1452/Dockerfile#L75

Reply:
Actually we can likely remove gnupg. Let me try.